src/mono/sample/wasm/wasm.mk

```diff
@@ -39,7 +39,7 @@ run-browser:
 	fi
 
 run-console:
-	cd bin/$(CONFIG)/AppBundle && $(V8_PATH) --stack-trace-limit=1000 --single-threaded --expose_wasm $(MAIN_JS) -- $(DOTNET_MONO_LOG_LEVEL) --run $(CONSOLE_DLL) $(ARGS)
```

Comment:
This broke browser-bench's `make run` for console version. Why was `--run $(CONSOLE_DLL)` removed? Do I need to update browser-bench's Makefile somehow?